packages/testlab/src/test-sandbox.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/testlab
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {tmpdir} from 'os';
+import {createHash} from 'crypto';
+import {resolve, join, parse} from 'path';
+import * as util from 'util';
+import {mkdtempSync, mkdir as mkdir_, copyFile as copyFile_} from 'fs';
```

Comment:
Have you considered `import * as fs from 'fs'` to avoid underscored variable names?

```ts
import * as fs from 'fs';

const mkdir = promisify(fs.mkdir);
```

Reply:
Do we have a convention for this? I just figured it might be better to explicitly state the imports being used from fs. I can change to `import * as fs from 'fs';` if that's better.